packages/compass-connections/src/components/connection-list/connection-list.tsx

```diff
@@ -127,26 +163,38 @@ function ConnectionList({
 }): React.ReactElement {
   const [recentHeaderHover, setRecentHover] = useState(false);
 
+  const { theme } = useTheme();
+
+  const useNewSidebar = process?.env?.COMPASS_SHOW_NEW_SIDEBAR === 'true';
```

Comment:
By the way, I know we’re doing this elsewhere but I think the `?.` are unnecessary – if `process` is defined, `process.env` will be defined and an object as well. We could either use plain property accesses or add `globalThis?.` at the front of the expression to actually enable this code to run in non-Node.js/Electron environments.

Reply:
They are unnecessary for non node envs too as all plugins are passed through webpack and it will provide process to the modules code. Also worth noting that it can statically replace them with values (where applicable) when optional chaining is used, but doing globalThis?.process will actually will trip it and opt out of static replacement (but maybe webpack got better at this in latest versions)

Reply:
I just made this consistent with what we have elsewhere so it is easy to find.